content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding.md

```diff
@@ -89,11 +89,11 @@ You should receive output similar to the following.
   \____| |_| |_| /_/   \_\ |___| |_| \_|  \____|   |_|   |_____|
 chainctl: Chainguard Control
 
-GitVersion:    0.1.39
-GitCommit:     98f4b3bc8a1ef111777f797e8248b737643eedc6
+GitVersion:    0.1.44
```

Comment:
```suggestion
GitVersion:    <semver version>
```

Reply:
Worth updating this with the generic placeholders now, or should we do it across every page in a separate PR?